packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -420,7 +448,9 @@ class _TooltipState extends State<Tooltip> with SingleTickerProviderStateMixin {
     );
     _entry = OverlayEntry(builder: (BuildContext context) => overlay);
     overlayState.insert(_entry!);
-    SemanticsService.tooltip(widget.message);
+    if (widget.message != null) {
+      SemanticsService.tooltip(widget.message!);
+    }
```

Comment:
The tooltip still needs to be somehow announced to the semantics layer for accessibility purposes even when a widget is given instead of a message. How do you plan to do that?

Reply:
We can deprecate the message property and introduce the semanticLabel property like other widgets and use it for accessibility purposes. Also, I was thinking that widgets also have semantics property but not sure how they will react in real-time.

Reply:
Deprecating the message property is probably unnecessarily breaking and makes the Tooltip for the simple cases harder to use.

We still need to figure out what to do about the semantics, though.